src/components/ArtifactDetailedInfo.tsx

```diff
@@ -91,177 +279,103 @@ const ArtifactDetailedInfoKojiBuild: React.FC<
-                        <DescriptionListDescription>
-                            <a
-                                className={styles['buildInfoCommitHash']}
-                                href={mkLinkPkgsDevelFromSource(
-                                    build.source,
-                                    instance,
-                                )}
-                                target="_blank"
-                                rel="noopener noreferrer"
-                            >
-                                {mkCommitHashFromSource(build.source)}
-                            </a>
-                        </DescriptionListDescription>
-                    </DescriptionListGroup>
-                    <DescriptionListGroup>
-                        <DescriptionListTerm>Build owner</DescriptionListTerm>
-                        <DescriptionListDescription>
-                            <a
-                                href={mkLinkKojiWebUserId(
-                                    build.owner_id,
-                                    instance,
-                                )}
-                                target="_blank"
-                                rel="noopener noreferrer"
-                            >
-                                {build.owner_name}
-                            </a>
-                        </DescriptionListDescription>
-                    </DescriptionListGroup>
-                    <DescriptionListGroup>
-                        <DescriptionListTerm>Committer</DescriptionListTerm>
-                        <DescriptionListDescription>
-                            {build.commit_obj?.committer_name || 'n/a'}
-                            &nbsp;&lt;
-                            {build.commit_obj?.committer_email || 'n/a'}
-                            &gt;
-                        </DescriptionListDescription>
-                    </DescriptionListGroup>
-                    <DescriptionListGroup>
-                        <DescriptionListTerm>
-                            Build completed
-                        </DescriptionListTerm>
-                        <DescriptionListDescription
-                            className={styles['buildInfoTimestamp']}
-                        >
-                            {buildTimeWithTz}
-                        </DescriptionListDescription>
-                    </DescriptionListGroup>
-                    <DescriptionListGroup>
-                        <DescriptionListTerm>Commit time</DescriptionListTerm>
-                        <DescriptionListDescription
-                            className={styles['buildInfoTimestamp']}
-                        >
-                            {commitTimeWithTz}
-                        </DescriptionListDescription>
-                    </DescriptionListGroup>
-                </DescriptionList>
+                <NoData show={_.every([!haveData, !loadingCurrentState])} />
```

Comment:
This seems unnecessarily contrived. Why not just this?
```suggestion
                <NoData show={!haveData && !loadingCurrentState} />
```